packages/tools/pie-css/test/scss/unit/validity.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+import cssValidator from 'w3c-css-validator';
+import { compileCss } from '../../../utilities/compileCss';
```

Comment:
Would this path work?

```suggestion
import { compileCss } from './utilities/compileCss';
```